Easydict/Feature/PerferenceWindow/DisableAutoSelectTextViewController/EZDisableAutoSelectTextViewController.m

```diff
@@ -78,6 +78,18 @@ - (void)setup {
     }];
 }
 
+- (void)setupAppModelList {
+    self.appModelList = [[NSMutableArray alloc] init];
+    NSArray<EZAppModel *> *allAppModelList = [EZLocalStorage.shared selectTextTypeAppModelList];
+    NSWorkspace* workspace = [NSWorkspace sharedWorkspace];
+    for (EZAppModel *appModel in allAppModelList) {
+        NSURL *appURL = [workspace URLForApplicationWithBundleIdentifier:appModel.appBundleID];
+        if (appURL) {
+            [self.appModelList addObject:appModel];
+        }
```

Comment:
I debugged the old code, found that when uninstalling the App in the disabled list, `appURL` is nil , which indeed causes the following method to crash 😓

```objc
NSBundle *appBundle = [[NSBundle alloc] initWithURL:appURL];
```

https://github.com/tisfeng/Easydict/blob/dev/Easydict/Feature/PerferenceWindow/DisableAutoSelectTextViewController/EZAppCell.m#L65

Reply:
Yes, it's very corner case. 😅